OrbitCore/CaptureData.h

```diff
@@ -23,7 +24,8 @@ class CaptureData {
         process_{std::move(process)},
         selected_functions_{std::move(selected_functions)},
         callstack_data_(std::make_unique<CallstackData>()),
-        selection_callstack_data_(std::make_unique<CallstackData>()) {
+        selection_callstack_data_(std::make_unique<CallstackData>()),
+        sampling_profiler_{std::make_shared<SamplingProfiler>(process_)} {
```

Comment:
Is there a reason why this is not in DataManager? It seems like this should live separately, even though it uses the capture data and is tied to capture state (it uses capture process not selected process).

Reply:
The idea was that the lifetime is completely bound by the one of capture data (at least in theory).

Reply:
On the other hand we do not want to serialize it. And the lifetime is actually a bit different - the sampling profiler data can only be calculated once capture is done.

Reply:
discussed offline - decided to keep it here because we want it to be freed when capture data is deallocated.